src/app/views/communities/view/overview/_nav/edit/edit.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+	<nav>
+		<template v-if="Screen.isMobile">
+			<div class="-item active" @click="isNavExpanded = !isNavExpanded">
+				<span class="-menu-icon">
+					<app-jolticon icon="menu" class="-jolticon middle" />
+				</span>
+				<span class="-label">
+					{{ activeItem }}
+				</span>
+			</div>
+		</template>
+
+		<div
+			v-if="!Screen.isMobile || isNavExpanded"
+			:class="{
+				'-mobile-nav-container fill-darker anim-fade-in-up': Screen.isMobile,
+			}"
+		>
+			<ol>
+				<app-community-perms
+					:community="community"
+					required="community-media,community-details"
```

Comment:
is media perm needed? the media is currently thumbnail + header which are editable from the page header where you are.

Reply:
Ah, I see.. is it because the thumbnail is squashed into the details page on mobile? Feels kind of odd putting it in details tho, but there isn't enough meat to justify a design section either. hmm...

Reply:
Yeah, didn't want to do that especially because it's mobile only.
We can reorganize that later when we have themes.